src/main/resources/db/migration/V1_2__create_table_oppfolgingstilfelle_bit.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE TILFELLE_BIT
+(
+    id                           SERIAL               PRIMARY KEY,
+    uuid                         VARCHAR(50)          NOT NULL UNIQUE,
+    created_at                   timestamptz          NOT NULL,
+    intruffet                    timestamptz          NOT NULL,
+    personident                  VARCHAR              NOT NULL,
```

Comment:
VARCHAR(11)?

Reply:
Endret til varchar(9), varchar(11), og text for tags.